### PR TITLE
Add more translation for zh-cn

### PR DESCRIPTION
**Pull request by recolic**
This pull request is adding more translation for zh-cn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,9 @@ In order to understand all of these new features, TerraFirmaCraft comes with a d
...
  - In addition, we request that you give us the right to change the license in the future.
  - Import & use the project's Code Style. (Recommend using Intellij as that's what our code style xml is based on)
-- Before creating a Pull Request, run `gradlew updateLicenses`. This will apply the correct license header to all project files.
-- We use python for data and asset generation, among other things. In order to run the generation scripts (all found in `/resources`), you will need to install the python modules `mcresources`, `pillow`, `nbtlib`, and `Levenshtein` (see `resources/requirements.txt`).
+ - Before creating a Pull Request, run `gradlew updateLicenses`. This will apply the correct license header to all project files.
+ - We use python for data and asset generation, among other things. In order to run the generation scripts (all found in `/resources`), you will need to install the python modules `mcresources`, `pillow`, `nbtlib`, and `Levenshtein` (see `resources/requirements.txt`).
+ - This mod depends on [Patchouli](https://www.curseforge.com/minecraft/mc-mods/patchouli). Your build will crash Minecraft silently without this.
```
Comment (alcatrazEscapee) [inline README.md:32]: It's not really silent, given that it both declares the mod as a runtime dependency in dev, will give you a large explicit error about a missing dependency when you run it in prod, and marks it as a mandatory dependency in CurseForge.

**Comment by alcatrazEscapee** [comment]
Merged in 775826e5580dca86356f008ef2befb948856bb32, with resource generation ran. If you're interested in future, [this comment](https://github.com/TerraFirmaCraft/TerraFirmaCraft/blob/775826e5580dca86356f008ef2befb948856bb32/resources/generate_book.py#L1-L70) explains both how to use our resource generation and setup a testing environment.

File: README.md (hunk 1 above)
Comment (recolic) [reply README.md:32]: I can see the warning while installing TFC without dependency only is the TFC jar is downloaded from curseforge. 
I tried to build it by my self, it crashes at exactly the same location( method not found), but there is no warning at all.. I'm not sure if I'm hitting some bug, and that's why I'm adding a warning here..

**Comment by recolic** [comment]
Thanks for your information! However, this is a draft PR, and I only translated 60% entries ... I will raise another PR after translating 100% entries. 

> Merged in [775826e](https://github.com/TerraFirmaCraft/TerraFirmaCraft/commit/775826e5580dca86356f008ef2befb948856bb32), with resource generation ran. If you're interested in future, [this comment](https://github.com/TerraFirmaCraft/TerraFirmaCraft/blob/775826e5580dca86356f008ef2befb948856bb32/resources/generate_book.py#L1-L70) explains both how to use our resource generation and setup a testing environment.

**Pull request closed without merging**